IBPSA/Fluid/HeatPumps/ModularReversible/UsersGuide.mo

```diff
@@ -0,0 +1,571 @@
+  First, the <a href=\"modelica://IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.ConstantCarnotEffectiveness\">
+  IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.ConstantCarnotEffectiveness</a> model
+  uses the same equations as the Carnot models, i.e.  <a href=\"modelica://IBPSA.Fluid.HeatPumps.Carnot_y\">
+  IBPSA.Fluid.HeatPumps.Carnot_y</a>.
+  Second, the <a href=\"modelica://IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.TableData2D\">
+  IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.TableData2D</a> provides performance data
+  based on different condenser outlet and evaporator inlet temperatures using 2D-tables.
+</p>
+<p>
+  Two additional modules exist in the AixLib library.
+  These approaches require the use of the SDF-library, as
+  the compressor speed influences the model output as a third
+  dimension. Currently, tables with more than two dimensions are not supported in
+  the Modelica Standard Library.
+  The first approach is similar to <a href=\"modelica://IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.TableData2D\">
+  IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.TableData2D</a>
+  approach buts adds the 3rd dimension of compressor speed.
+  The second approach is based on white-box stationary Python models
+  for closed-loop refrigerant cycles. The model has been empirically
+  validated and can take up to n-dimensions.
+  If your simulation aim requires more detailed data, be sure
+  to check out the models in the AixLib.
+</p>
+
+
+<h4>Parameterization and naming</h4>
+<p>
+  The refrigerant cycle models will output
+  varying heat flow rates and electrical power consumptions.
+  This is based on the fact that refrigerant cycle performance
+  depend heavily on the boundary conditions.
+</p>
+<p>
+  Still, the user needs to size the device or the system
+  according to some reference points. Accordingly, we follow
+  the basic IBPSA concept for nominal conditions, explained in
+  detail here:
+  <a href=\"modelica://IBPSA.Fluid.UsersGuide\">IBPSA.Fluid.UsersGuide</a>
+</p>
+<p>
+  The nominal heat flow rate of the device for its main
+  usage is called <code>QUse_flow_nominal</code>.
+  For heat pumps, it is the nominal
+  condenser heat flow rate <code>QCon_flow_nominal</code>.
+  For chillers, it is the nominal
+  evaporator heat flow rate <code>QEva_flow_nominal</code>.
+  This nominal heat flow rate is only valid at the
+  nominal conditions of the
+</p>
+<ul>
+<li>
+  condenser temperature <code>TCon_nominal</code>,
+</li>
+<li>
+  evaporator temperature <code>TEva_nominal</code>,
+</li>
+<li>
+  condenser temperature difference<code>dTCon_nominal</code>,
+</li>
+<li>
+  evaporator temperature difference<code>dTEva_nominal</code>,
+</li>
+<li>
+  condenser mass flow rate <code>mCon_flow_nominal</code>,
+</li>
+<li>
+  evaporator mass flow rate <code>mCon_flow_nominal</code>, and
+</li>
+<li>
+  compressor speed <code>y_nominal</code>.
+</li>
+</ul>
```

Comment:
Can you add a short example of when to set `y_nominal` different from 1, and what data do get scaled in what manner (`PEle_nominal`, `mCon_flow_nominal?`, ...)

Reply:
I added a detailed explanation to the UserGuide with an example for a possible application. Is this sufficient or do you want a concrete .mo example?